src/app/pages/authentication/login/login.component.ts

```diff
@@ -0,0 +1,57 @@
+import { Component, OnInit } from '@angular/core';
+import { FormControl, FormGroup, Validators } from '@angular/forms';
+import { AuthService } from '../../../core/service/auth.service';
+import { Router } from '@angular/router';
+import { ToastService } from 'src/app/core/service/toast.service';
+
+@Component({
+  selector: 'app-login',
+  templateUrl: './login.component.html',
+  styleUrls: ['./login.component.css'],
+})
+export class LoginComponent implements OnInit {
+  loginForm: FormGroup;
+
+  constructor(
+    private authService: AuthService,
+    private router: Router,
+    private toastService: ToastService
+  ) {
+    this.loginForm = new FormGroup({
+      email: new FormControl('', [
+        Validators.required,
+        Validators.pattern(/^[a-zA-Z0-9._%+-]+@[a-zA-Z0-9.-]+\.[a-zA-Z]{2,}$/),
+      ]),
+      password: new FormControl('', [Validators.required]),
+    });
+  }
+
+  ngOnInit(): void {}
+
+  goToRegistration() {
+    this.router.navigate(['/registration']);
```

Comment:
without writing a separate function you can also use routerLink in html template

Reply:
@aky-wtag vai resolved